index.html

```diff
@@ -74,18 +74,19 @@ <h1>Find all discounts </h1>
         </div>
     </div>
         
-    <div class="container mt-5 mb-5">
-        <div class="row text-center d-flex align-content-center flex-wrap" style="height: 150px; width: 100%; display:block; background-image: radial-gradient(circle, #eab816, #c0d45f, #a8e6a2, #b2efd7, #def2f1);">
+    <div class="container mt-5 mb-5" style="max-width: 100%;">
+        <div class="row text-center d-flex align-content-center flex-wrap" style="height: 150px; display:block; background-image: radial-gradient(circle, #eab816, #c0d45f, #a8e6a2, #b2efd7, #def2f1);">
                 <div class="col-xl-12"> 
-                    <p class="font-italic font-weight-bold text-center">Select your favorite market and browse through the offers in your region</p>
+                    <p class="font-italic font-weight-bold text-center">Find the supermarket near your place and get your favourite product with discount</p>
                 </div>
                 <div class="col-xl-12">
                     <h2> <a href="supermarkets.html" class="badge" style="color: white; background-color: #2B7A78">Search now</a></h2>
                 </div>
         </div>
     </div>
     
-    <div class="container mt-5 mb-5" id="products" style="max-width: 95%; display: block;">
+    
+    <div class="container mt-5 mb-5" id="products" style="max-width: 95%; display:">
```

Comment:
I think that style should be broken. At least `display:` part.
And again: you can move it to some css class.